code/campaign/src/test/java/com/adobe/marketing/mobile/campaign/FullScreenMessageTests.java

```diff
@@ -379,6 +392,95 @@ public void showMessage_Should_Call_setLocalAssetsMap_with_a_Non_Empty_Map_WhenA
         });
     }
 
+    // bundled asset caching
+    @Test
+    public void showMessage_ShouldCacheBundledAssets_When_RemoteAssetFailedToBeCachedAndLocalAssetPresentInConsequenceDetails() {
+        // setup
+        setupServiceProviderMockAndRunTest(() -> {
+            Mockito.when(mockCacheService.get(anyString(), eq("http://asset1-url00.jpeg"))).thenReturn(null);
+            Map<String, String> expectedMap = new HashMap<>();
+            expectedMap.put("file:///android_asset/http://asset1-url00.jpeg", "campaign/messages");
```

Comment:
Not sure I understand this path. The protocol for locally cached asset is already `file:///`, why do you have `http://` for local asset? It's confusing, let's remove it.

Reply:
this was actually an issue with the url being used when storing a cached image